.gitignore

```diff
@@ -4,3 +4,4 @@ pelicanconf.pyc
 balab/
 lists/
 output/
+.DS_Store
```

Comment:
Please do not commit system files. Run `git diff` before committing to see what will end in the repo.